src/components/Channel/Channel.js

```diff
@@ -174,7 +174,7 @@ const ChannelInner = ({
 
         if (
           mainChannelUpdated &&
-          e.message.user.id !== chatContext.client.userID
+          e.message.user.id !== chatContext.client?.userID
```

Comment:
Since we reverted https://github.com/GetStream/stream-chat-react/pull/461/commits/3d0ced6059416b30d3f6482cd9087a0bcda6fc0b#diff-88bec6beae84369c0376b56b3bb88fe1L14, this shouldn't be needed.

Reply:
Same for all the other changes in this file.